sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.net8.0.cs

```diff
@@ -5606,7 +5798,7 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
     }
     public partial class UriSigningKeyProperties : Azure.ResourceManager.Cdn.Models.FrontDoorSecretProperties, System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Cdn.Models.UriSigningKeyProperties>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Cdn.Models.UriSigningKeyProperties>
     {
-        public UriSigningKeyProperties(string keyId, Azure.ResourceManager.Resources.Models.WritableSubResource secretSource) { }
+        public UriSigningKeyProperties(string keyId, Azure.ResourceManager.Resources.Models.WritableSubResource secretSource, string secretVersion) { }
```

Comment:
is this change introduced by our customization code?
This looks incorrect, because the parameters in a model's ctor will be assigned to its properties.
This ctor adds a parameter, but this model does not add any property, therefore this looks incorrect.

Reply:
I see, the new property was introduced in the base type, therefore we have a change here.
If the apicompat does not report this, maybe this is fine.